Makefile

```diff
@@ -23,6 +23,10 @@ trigger: start-deps
 	. ./.envrc && yarn tsnd --respawn --files -r tsconfig-paths/register -r src/services/tracing.ts \
 		src/servers/trigger.ts | yarn pino-pretty -c -l
 
+start-servers-ci:
```

Comment:
When is this supposed to be executed?

Reply:
Its executed at the start of an integration test suite  using the functions exported from test/helpers/integration-server.ts